docs/reuse/common-blocks.md

```diff
@@ -0,0 +1,44 @@
+Start: Install Pebble binary
+
+1. Visit the [latest release page](https://github.com/canonical/pebble/releases/latest) to get the latest tag.
+2. Run the following commands to download the file. Make sure to replace `v1.12.0` with the latest tag and `amd64` with your architecture.
+    ```bash
+    wget https://github.com/canonical/pebble/releases/download/v1.12.0/pebble_v1.12.0_linux_amd64.tar.gz
+    tar zxvf pebble_v1.12.0_linux_amd64.tar.gz
+    sudo mv pebble /usr/local/bin/
+    ```
+3. Extract the contents of the downloaded file by running:
+    ```bash
+    tar zxvf pebble_v1.12.0_linux_amd64.tar.gz
+    ```
+4. Install the Pebble binary. Make sure it's included in your system's `PATH` environment variable.
```

Comment:
It's a bit unclear what "it" refers to here. Perhaps (Pebble binary itself, or `/usr/local/bin`?). Maybe:

```suggestion
4. Install the Pebble binary. Make sure the installation directory is included in your system's `PATH` environment variable.
```